man/docker-exec.1.md

```diff
@@ -7,6 +7,7 @@ docker-exec - Run a command in a running container
 # SYNOPSIS
 **docker exec**
 [**-d**|**--detach**]
+[**--detach-keys**[=*[]*]]
```

Comment:
@moxiegirl I think it's the same as some other flags, where you can use them with or without the `=` sign.